huaweicloud/resource_huaweicloud_identity_project_test.go

```diff
@@ -14,13 +14,14 @@ import (
 
 func TestAccIdentityV3Project_basic(t *testing.T) {
 	var project projects.Project
-	var projectName = fmt.Sprintf("ACCPTTEST-%s", acctest.RandString(5))
+	var projectName = fmt.Sprintf("cn-north-4_tf-acc-test-%s", acctest.RandString(5))
```

Comment:
hard coding is not recommend, please keep it as it is

Reply:
fixed